core/src/com/unciv/ui/screens/civilopediascreen/FormattedLine.kt

```diff
@@ -316,6 +296,28 @@ class FormattedLine (
         return table
     }
 
+    private fun renderExtraImage(labelWidth: Float): Table {
+        val table = Table(BaseScreen.skin)
+        fun getExtraImage(): Image {
+            if (ImageGetter.imageExists(extraImage))
+                return if (centered) ImageGetter.getDrawable(extraImage).cropToContent()
+                    else ImageGetter.getImage(extraImage)
+            val externalImage = ImageGetter.findExternalImage(extraImage)
+                ?: throw UncivShowableException("Extra image '[$extraImage]' not found")
+            return ImageGetter.getExternalImage(externalImage)
+        }
+        val image = getExtraImage()
+        // limit larger cordinate to a given max size
+        val maxSize = if (imageSize.isNaN()) labelWidth else imageSize
+        val (width, height) = if (image.width > image.height)
```

Comment:
This is a *nice trick*! I've only seen this is python/c#, didn't know you could do it in Kotlin as well!

6 years in and I'm still learning new things :)